spec/features/visitor/media/visitor_get_media_spec.rb

```diff
@@ -14,9 +14,8 @@
   scenario scenario do
     media = create(:media, slug: 'image')
 
-    expect{ visit media_path(media.slug) }.to raise_error(ActionController::RoutingError)
+    visit media_path(media.slug)
 
-    expect(page.status_code).to eq(303)
-    expect(page.current_url).to eq(media.file.url)
+    expect(URI.parse(current_url).path).to eq(media.file.url)
```

Comment:
```
expected: "/system/medias/files/000/002/142/original/demo.jpg"
got: "http://localhost:5050/system/medias/files/000/002/142/original/demo.jpg"
```
likely when using filesystem options Paperclip record the file path without hostname